php/commands/sidebar.php

```diff
@@ -73,6 +73,204 @@ public function list_( $args, $assoc_args ) {
 		$formatter->display_items( $wp_registered_sidebars );
 	}
 
+	/**
+	 * Reset one or all sidebars.
+	 *
```

Comment:
Also, can you include one or more examples in the new pattern we've established?